.github/CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 /*/manifest.json                         @DataDog/documentation @DataDog/agent-integrations
 
 # Community partners
+/agora_analytics/                        @Agora zhangpeizhe@agora.io      
```

Comment:
please add @DataDog/marketplace-review to the codeowners file

Reply:
Added